pkg/dataload/constants.go

```diff
@@ -21,4 +21,6 @@ const (
 	DATALOAD_DEFAULT_IMAGE = "registry.cn-hangzhou.aliyuncs.com/fluid/fluid-dataloader"
 	DATALOAD_SUFFIX_LENGTH = 5
 	ENV_DATALOADER_IMG     = "DATALOADER_IMG"
+	DATALOAD_ALLUXIO_CHART = "alluxio"
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

Reply:
`alluxio` appear many time, can it be unified into one place ?

Reply:
thx, removed and replaced by common constant